maths-club-pack/content/student/colouring-maps.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Colouring Maps
+type: problem
+printOrder: n/a
+added: 2021-01
```

Comment:
You might want to change both problems `added` to `2021-03`, just to make sure they take top priority in the app